### DISCUSSION TITLE
large pdf problem

### DISCUSSION THREAD

**Discussion by ahmadsaddin** [FlowiseAI/Flowise#305] [Q&A]
hello there i would like to ask if anyone can help.
the problem is that i have a nearly 60 mb pdf file and when i uploaded it i got an error that the pdf file is too big, does anyone know how can i fix this problem?
![pic1](https://github.com/FlowiseAI/Flowise/assets/109594904/9ff2cfc3-7144-4a7b-876e-d3bb83849ee7)

**Comment by HenryHengZJ** [comment]
here - https://github.com/FlowiseAI/Flowise/issues/96

**Reply by ahmadsaddin** [reply]
thank you so much for your help
